bases/base_mysql.go

```diff
@@ -101,6 +101,10 @@ func generateMysqlTypes(obj map[string]map[string]string, depth int, jsonAnnotat
 			annotations = append(annotations, fmt.Sprintf("json:\"%s\"", key))
 		}
 
+		if dbAnnotation {
+			annotations = append(annotations, fmt.Sprintf("db:\"%s\"", key))
```

Comment:
What is the function of `dbAnnotation` tag ?

Reply:
```suggestion
			annotations = append(annotations, fmt.Sprintf("db:\"%s\"", key))
```
new sql orm tag in our framework，here https://github.com/huandu/go-sqlbuilder

Reply:
I got it. But, you should add the corresponding parameters to fuckdb.json and give it a default value (false).

Reply:
ok thanks, i will fix it